code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -437,7 +437,7 @@ obj/effect/sweatsplash/proc/splash()
 		tetsuo = TRUE
 	var/mob/living/carbon/M = A.affected_mob
 	ownermind = M.mind
-	sizemult = min(max(1.1, (0.5 + A.properties["stage_rate"] / 10)), 2.5)
+	sizemult = clamp((0.5 + A.properties["stage_rate"] / 10), 1.1, 2.5)

```

Comment:
I believe you need the CLAMP macro which is in full caps

Reply:
Aw fuck i lowercased it, lemme webedit and then go back to sleep